### DISCUSSION TITLE
Is it possible to delete environment tab from GitHub repo?

### DISCUSSION THREAD

**Discussion by usernamehw** [community/community#23253] [Other Feature Feedback, Questions, & Ideas]
<p>I accidentally published the repo to GitHub pages (not a problem in itself).</p>
<p>Then deleted/unpublished.</p>
<p>Now there’s an Environment Tab that stands out (unlike all of my other repositories)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/af8adc1b7298cf2ea6f4cd70f5f94d23032e066b.png" data-download-href="/uploads/short-url/p2V3Uq62B71thPAe2BCaIRTES8j.png?dl=1" title="Screenshot (218).png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/9638156/181075934-c25d6e91-f022-408f-a32b-f7dbb9f6bd51.png" alt="Screenshot (218).png" data-base62-sha1="p2V3Uq62B71thPAe2BCaIRTES8j" width="690" height="37" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/af8adc1b7298cf2ea6f4cd70f5f94d23032e066b_2_690x37.png, https://aws1.discourse-cdn.com/github/optimized/1X/af8adc1b7298cf2ea6f4cd70f5f94d23032e066b_2_1035x55.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/af8adc1b7298cf2ea6f4cd70f5f94d23032e066b_2_1380x74.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/af8adc1b7298cf2ea6f4cd70f5f94d23032e066b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (218).png</span><span class="informations">1486×80 7.96 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Want to remove it.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/usernamehw">@usernamehw</a>,</p>
<p>Sorry to say that it’s not currently possible to remove deployments from the environment tab. However, it is a feature request that we’re tracking and I’ve added your +1.</p>
